test/validation/api/scheduler/scheduler.c

```diff
@@ -988,6 +986,46 @@ static void scheduler_test_create_group(void)
 	CU_ASSERT_FATAL(odp_schedule(NULL, wait_time) == ODP_EVENT_INVALID);
 }
 
+static void scheduler_test_create_group(void)
+{
+	odp_thrmask_t mask;
+	odp_schedule_group_t group;
+
+	odp_thrmask_zero(&mask);
+	odp_thrmask_set(&mask, odp_thread_id());
+	group = odp_schedule_group_create("create_group", &mask);
+
+	CU_ASSERT_FATAL(group != ODP_SCHED_GROUP_INVALID);
+
+	test_group_basic(group);
+}
+
+static void scheduler_test_group_param_init(void)
+{
+	odp_schedule_group_param_t param;
+
+	memset(&param, 0xff, sizeof(param));
+	odp_schedule_group_param_init(&param);
+	CU_ASSERT(param.cache_stash_hints.common.regions.all == 0);
+	CU_ASSERT(param.cache_stash_hints.num == 0);
+}
+
+static void scheduler_test_create_group_2(void)
```

Comment:
Could also add a similar test which enables cache stashing. Assuming `odp_schedule_group_create_2()` should always pass regardless of if cache stashing is supported, as those are only hints.

Reply:
Added some cache stashing hints, the creation should not fail because of inability to handle them.